src/app/beer_garden/api/http/exceptions.py

```diff
@@ -46,16 +46,6 @@ class AuthenticationFailed(BaseHTTPError):
     reason: str = "Authentication Failed"
 
 
-class InvalidToken(BaseHTTPError):
```

Comment:
I decided to just get rid of these because the they were only being used in one place and with the refactor required for this branch I felt its usage was more confusing rather than less.

What the usage would have looked like with these here:
```python
except InvalidTokenException:
    raise InvalidToken
except ExpiredTokenException:
    raise ExpiredToken
```

And with these removed:

```python
except InvalidTokenException:
    raise RequestForbidden(reason="Authorization token invalid")
except ExpiredTokenException:
    raise AuthorizationRequired(reason="Authorization token expired")
```

I felt like that second one makes it more obvious what the HTTP error that's ultimately getting returned will be.